SyncPro.Adapters.WindowsFileSystem/WindowsFileSystemAdapter.cs

```diff
@@ -175,7 +175,8 @@ public override void UpdateItem(EntryUpdateInfo updateInfo, SyncEntryChangedFlag
                 else if (updateInfo.Entry.Type == SyncEntryType.Directory)
                 {
                     Pre.Assert(!string.IsNullOrEmpty(newFullPath), "newFullPath != null");
-                    Directory.Move(fullPath, newFullPath);
+                    if (fullPath != newFullPath)
```

Comment:
This would be better written as string.equals.
```suggestion
                    if (!string.Equals(fullPath, newFullPath))
```

Reply:
Yes, that would be better.